test/lib/code_corps/github/event/installation/repos_test.exs

```diff
@@ -93,25 +107,23 @@ defmodule CodeCorps.GitHub.Event.Installation.ReposTest do
       assert GithubRepo |> Repo.aggregate(:count, :id) == 2
     end
 
-    @tag bypass: %{
-      "/installation/repositories" => {403, @forbidden},
-      "/installations/#{@app_github_id}/access_tokens" => {200, @access_token_create_response}
-    }
     test "returns installation as errored if api error" do
       installation = insert(:github_app_installation, github_id: @app_github_id, state: "initiated_on_code_corps")
-      {:error, %GithubAppInstallation{state: state}, %CodeCorps.GitHub.APIError{}} = Repos.process(installation)
+
+      with_mock_api(BadRepoRequest) do
+        {:error, %GithubAppInstallation{state: state}, %CodeCorps.GitHub.APIError{}} = Repos.process(installation)
+      end
```

Comment:
This is how we use the macro to mock out an api request.

With that done, the bypass tag is no longer needed. Our goal in this PR is to remove all existing bypass tags and switch to `with_mock_api`. 

While doing that, we will fill out the default `SuccessAPI` module with default responses.

Something to consider is that it may also make sense to define a default `FailureAPI` if we see enough code being repeated.